README.md

```diff
@@ -6,6 +6,7 @@
 </p>
 
 [![CircleCI](https://circleci.com/gh/logdna/logdna-go/tree/master.svg?style=svg)](https://circleci.com/gh/logdna/logdna-go/tree/master)
+[![Coverage Status](https://coveralls.io/repos/github/logdna/logdna-go/badge.svg?branch=mdeltito/LOG-6966)](https://coveralls.io/github/logdna/logdna-go?branch=mdeltito/LOG-6966)
```

Comment:
This should be the `master` branch so the badge shows up when it merges.  It will be blank until then, but that's ok

Reply:
ah good call, did not notice when copy/pasting

Reply:
fixed